recipes/dsdp/meta.yaml

```diff
@@ -0,0 +1,55 @@
+{% set version = "5.8" %}
+
+{% set variant = "openblas" %}
+
+
+package:
+  name: dsdp
+  version: {{ version }}
+
+source:
+  fn: DSDP{{ version }}.tar.gz
+  url: http://www.mcs.anl.gov/hs/software/DSDP/DSDP{{ version }}.tar.gz
+  sha256: 26aa624525a636de272c0b329e2dfd01a0d5b7827f1c1c76f393d71e37dead70
+  patches: dsdp_patch.diff
+
+build:
+  skip: true  # [win]
+  number: 200
+  features:
+    - blas_{{ variant }}
+
+requirements:
+  build:
+    - cmake
+    - {{ compiler("c") }}
+
+  host:
+    - blas 1.1 {{ variant }}
+    - openblas 0.2.20|0.2.20.*
+
+  run:
+    - blas 1.1 {{ variant }}
+    - openblas 0.2.20|0.2.20.*
+
+test:
+  source_files:
+    - bin/control1.dat-s
+    - bin/mcp100.dat-s
+    - bin/truss1.dat-s
+  commands:
+    - dsdp5 bin/control1.dat-s
+    - dsdp5 bin/mcp100.dat-s  
+    - dsdp5 bin/truss1.dat-s  
+    - conda inspect linkages -p ${PREFIX} dsdp  
+    - conda inspect objects -p ${PREFIX} dsdp  # [osx]
```

Comment:
FWIW the `conda inspect` commands are heading towards deprecation. Also `conda build` 3 does these inspections in much better detail. Though we can remove these in the feedstock.